yarn.lock

```diff
@@ -19,18 +19,18 @@
     semver "^5.5.0"
 
 "@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.7.5":
-  version "7.11.4"
```

Comment:
Can we separate this out as a different change from `pr-linter`? 
From what I understand, they're not linked.